doc/release_notes_0.7.rst

```diff
@@ -0,0 +1,382 @@
+.. _release_notes_0.7:
+
+ACRN v0.7 (Mar 2019)
+####################
+
+We are pleased to announce the release of Project ACRN version 0.7.
+
+ACRN is a flexible, lightweight reference hypervisor, built with
+real-time and safety-criticality in mind, optimized to streamline
+embedded development through an open source platform. Check out the
+:ref:`introduction` for more information.
+
+
+All project ACRN source code is maintained in the
+https://github.com/projectacrn/acrn-hypervisor repository and includes
+folders for the ACRN hypervisor, the ACRN device model, tools, and
+documentation.  You can either download this source code as a zip or
+tar.gz file (see the `ACRN v0.7 GitHub release page
+<https://github.com/projectacrn/acrn-hypervisor/releases/tag/v0.7>`_ or
+use Git clone and checkout commands:
+
+.. code-block:: bash
+
+   git clone https://github.com/projectacrn/acrn-hypervisor
+   cd acrn-hypervisor
+   git checkout v0.7
+
+The project's online technical documentation is also tagged to correspond
+with a specific release: generated v0.7 documents can be found at
+https://projectacrn.github.io/0.7/.  Documentation for the latest
+(master) branch is found at https://projectacrn.github.io/latest/.
+
+ACRN v0.7 requires Clear Linux OS version 28260 or newer.  Please follow the
+instructions in the :ref:`getting-started-apl-nuc`.
+
+Version 0.7 new features
+************************
+
+Enable cache QOS with CAT
+=========================
+
+Cache Allocation Technology (CAT) is enabled on Apollo Lake (APL)
+platforms, providing cache isolation between VMs mainly for real-time
+performance quality of service (QoS).  The CAT for a specific VM is
+normally set up at boot time per the VM configuration determined at
+build time. For debugging and performancing tuning, the CAT can also be
+enabled and configured at runtime by writing proper values to certain
+MSRs using the ``wrmsr`` command on ACRN shell.
+
+Support ACPI power key mediator
+===============================
+ACRN supports ACPI power/sleep key on the APL and KBL NUC platforms,
```

Comment:
Is this specific to APL and KBL or is it more a case of SBL vs. EFI?

Reply:
@NanlinXie ?

Reply:
@gvancuts It is for  UEFI platform, OpenSource feature.

Reply:
Thanks @NanlinXie , is therefore true that UP2 with SBL is not support but that UP2 running EFI will work (just like the NUCs)?

If so, I would suggest to reword this to indicate that we support these on EFI platforms.